hunting-buddy.lic

```diff
@@ -54,31 +54,36 @@ class HuntingBuddy
       duration = info[:duration]
       stop_on_skills = info['stop_on']
 
-      # Execute scripts to run before the hunt
-      before_actions.each do |before_action|
-        action_parts = before_action.split(' ')
-        script_name = action_parts.shift
-        wait_for_script_to_complete(script_name, action_parts)
-      end
+      #Execute scripts to run before the hunt
+      execute_actions(before_actions)
 
-      next if all_skills_at_cap?(stop_on_skills)
+      if all_skills_at_cap?(stop_on_skills)
```

Comment:
This means that if all skills are at cap, we will call `before` actions and then immediately `after` actions. Do you think we should instead check for skills at cap before the `before` actions are executed?